### DISCUSSION TITLE
How can the text input placeholder be changed from "Write a message..." ?

### DISCUSSION THREAD

**Discussion by kosso** [telegraf/telegraf#1588] [Q&A] [answered]
Is it possible to change the main text input placeholder text from "Write a message..." to something else ? 

It appears that possibly a bot in a channel I help Moderate may have done it during some setup process. Or is this related to 'hear'?

I'm currently learning to build with telegraf to create another bot for our channel. So if I can solve this placeholder issue, or how find out how on earth the change happened, that would be great.  Thanks!
![Screenshot 2022-03-10 at 23 39 16](https://user-images.githubusercontent.com/202782/157777587-e87bb74c-9d13-4d55-900d-217b8ca77853.png)

**Comment by hypernova7** [comment, marked as answer]
That feature only works with keyboard
```javascript
ctx.reply('text', {
  ...Markup
    .placeholder('Write a message...')
    .keyboard([
      Markup.button.text('I\'m button')
    ])
})
```
